NsqSharp/Utils/TlsConfig.cs

```diff
@@ -14,7 +14,7 @@ public class TlsConfig
         /// </summary>
         public TlsConfig()
         {
-#if NETFX_3_5 || NETFX_4_0
+#if NETFX_4_0
             MinVersion = SslProtocols.Tls;
 #else
             MinVersion = SslProtocols.Tls12;
```

Comment:
We are supposed to start supporting Tls 1.2 ( only ) from end of April ( deadline is is 30th of June 2018, more here : https://blog.pcisecuritystandards.org/migrating-from-ssl-and-early-tls ) so you may need to revisit this logic later on.